application/app_config.py

```diff
@@ -4,7 +4,7 @@
 
 
 class AppConfig(object):
-    PROJECT_DIR = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "project_dir"))
+    PROJECT_DIR = os.path.realpath(os.path.join(os.path.expanduser('~'),"Documents" , "TrafficAnalysis", "project_dir"))
```

Comment:
I'll make the Application name a variable for now, until we settle on a finalized name. I'll set it as SantosGUI now. Note other code will create the directory in the process of creating a project. In addition, opening another project overwrites this path, so it should be possible to load old projects from any directory.